### DISCUSSION TITLE
Hi everyone :)

### DISCUSSION THREAD

**Discussion by Metrix1010** [OpenMined/courses#151] [Introduce Yourself]
Hi everyone,

I am Laura from Alberta Canada. I just finished Lesson 4 and have been at a loss for words since Lesson 2. The course material, quality of presentation, excellent explanations, analogies and examples are so helpful.  I am very excited about what structured transparency will mean for privacy enhancing technologies! So many opportunities will arise from this framework to transform the incentive structure to one that serves humanity. How do you choose just one area to focus on?  I also cannot stop thinking about data as fire, and Homomorphic Encryption as clothing but that's just the anthropologist in me.  If you are new to the OpenMined community then welcome! Pull up a rock and let's make some s'mores.

**Comment by yemikifouly** [comment]
Welcome to the discussion board Laura! ❤️ S'mores sound good! 😀
